jsonnet/main.jsonnet

```diff
@@ -0,0 +1,3 @@
+local app = import 'lib/thanos-receive-controller.libsonnet';
+
+{ ['thanos-receive-controller-' + name]: app.thanos.receiveController[name] for name in std.objectFields(app.thanos.receiveController) }
```

Comment:
we can omit the `thanos-receive-controller-` prefix here, since this is assumed for this repo (: